.github/workflows/cleanup-dev-release.yml

```diff
@@ -0,0 +1,42 @@
+name: "Cleanup Dev Release"
+
+on:
+  pull_request:
+    types: [unlabeled, closed]
+
+jobs:
+  cleanup:
+    # Only run when the 'create-dev-release' label is removed or PR is closed
+    if: |
+      (github.event.action == 'unlabeled' && github.event.label.name == 'create-dev-release') ||
+      (github.event.action == 'closed' && contains(github.event.pull_request.labels.*.name, 'create-dev-release'))
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+
+    steps:
+      - name: "Extract Branch Name"
```

Comment:
This is still inconsistent with the other action. Make them the same. It doesn't have this step at all and just does it inline in the payload